flink-end-to-end-tests/test-scripts/test_pyflink.sh

```diff
@@ -223,13 +223,23 @@ function read_msg_from_kafka {
 }
 
 function cat_jm_logs {
-     local log_file_name=${3:-standalonesession}
-     cat $FLINK_LOG_DIR/*$log_file_name*.log
+  cat_logs_in_order "standalonesession"
 }
 
 function cat_tm_logs {
-	local logfile="$FLINK_LOG_DIR/flink*taskexecutor*log"
-	cat ${logfile}
+  cat_logs_in_order "flink*taskexecutor"
+}
+
+function cat_logs_in_order {
```

Comment:
Essentially, I pinged you on that change to have another look whether it's reasonable since I introduced a new more general function to cover the two implementations above